js/src/personal/jionghao/components/About.tsx

```diff
@@ -0,0 +1,50 @@
+import { Title, Text, Image } from '@mantine/core'
+import styles from './About.module.css'
+
+export function About() {
```

Comment:
nit: You might want to make each page its own folder under jionghao. Components should likely be things used between pages.

Reply:
ok got it